Mage/Module.cs

```diff
@@ -67,28 +83,46 @@ public void SetupStaticData(Action<Exception> cb)
 		}
 
 
-		//
-		protected virtual string CommandPrefix
-		{
-			get { return null; }
-		}
+		// The module name as defined on the remote MAGE server
+		protected abstract string CommandPrefix { get; }
```

Comment:
yes but current mage (JS) doesn't enforce these things either
if i have no usercommands I shouldn't have to care about these values
@nullorvoid @kefniark @ronkorving comments welcome